ci-operator/config/maistra/xns-informer/maistra-xns-informer-maistra-2.5.yaml

```diff
@@ -0,0 +1,40 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.5"
```

Comment:
This will fail, as we don't have a builder image for 2.5 yet.

Reply:
`quay.io/maistra-dev/maistra-builder:2.5` is already available